meta-mbl-apps/recipes-support/swupdate/swupdate-headers_2019.11.bb

```diff
@@ -0,0 +1,21 @@
+# Copyright (c) 2019 Arm Limited and Contributors. All rights reserved.
+#
+# SPDX-License-Identifier: MIT
+
+require recipes-support/swupdate/swupdate_${PV}.bb
+require swupdate-append-common.inc
+
+SUMMARY = "Header files for building handlers for swupdate"
+DESCRIPTION = "${SUMMARY}"
+FILESEXTRAPATHS_prepend = "${THISDIR}/../../../../meta-swupdate/recipes-support/swupdate/swupdate:"
```

Comment:
I'm not entirely sure if this is needed. I'm expecting that the `require` will bring the `FILESPATH` from the required recipe.
If not, can you try to use one of these two options instead of "../'s"?
`FILESEXTRAPATHS_prepend = "meta-swupdate/recipes-support/swupdate/swupdate:"`
or
`FILESEXTRAPATHS_prepend = "${OEROOT}/layers/meta-swupdate/recipes-support/swupdate/swupdate:"`

Reply:
> I'm expecting that the `require` will bring the `FILESPATH` from the required recipe.

It doesn't, unfortunately.

 > `FILESEXTRAPATHS_prepend = "meta-swupdate/recipes-support/swupdate/swupdate:"`

I tried that, but `FILESPATH` entries are absolute paths.

> `FILESEXTRAPATHS_prepend = "${OEROOT}/layers/meta-swupdate/recipes-support/swupdate/swupdate:"`

I thought I'd tried that, but I think maybe I had a typo in the path and thought it wouldn't work when in fact, it does... Or at least `bitbake -e swupdate-headers` tells me that `$OEROOT` has a reasonable value. 

Thanks for forcing me to try it again :-)

TBH, I still don't like this line even when using `${OEROOT}` - it still makes an assumption about where we keep our layers. I can't think of anything better though.

Reply:
https://github.com/ARMmbed/meta-mbl/pull/811/commits/231556c65ee51a13283f25ca8f8c658d83379bb0

Reply:
> > I'm expecting that the `require` will bring the `FILESPATH` from the required recipe.
> 
> It doesn't, unfortunately.
> 
Maybe this is only valid when you directly require/include a `.inc` file? Can you please try that?
`require recipes-support/swupdate/swupdate.inc` and remove the `FILESEXTRAPATHS_prepend` and see how it goes?

> TBH, I still don't like this line even when using `${OEROOT}` - it still makes an assumption about where we keep our layers. I can't think of anything better though.
Me neither. That's why I'm trying to explore other possibilities, but I'm not sure if you already tried what I'm proposing.

Reply:
> Maybe this is only valid when you directly require/include a .inc file? Can you please try that?

Even if `require`ing swupdate.inc without using an absolute path would work, I don't think that would be a good solution because the `SRCREV` comes from the swupdate `bb` file, not the `inc` file, and obtaining the correct `SRCREV` was one of the main motivations for this approach.